Portfolio/src/Wrapper.vue

```diff
@@ -0,0 +1,34 @@
+<script setup lang="ts">
+</script>
+
+<template>
+    <div class="wrapper">
```

Comment:
What kind of wrapper is this btw? The name is super generic so it is difficult for me to understand where exactly this wrapper is going to be used. So if u wrap the whole app with it u need to specify it :) Or if u wrap a link with it u need to specify it. And u do it through giving it a meaningful name

Reply:
Done